src/pushjack/apns.py

```diff
@@ -764,6 +776,346 @@ class APNSExpiredToken(namedtuple('APNSExpiredToken', ['token', 'timestamp'])):
     pass
 
 
+class HttpStatus(object):
+    OK = 200
+    BAD_REQUEST = 400
+    FORBIDDEN = 403
+    METHOD_NOT_ALLOWED = 405
+    GONE = 410
+    PAYLOAD_TOO_LARGE = 413
+    TOO_MANY_REQUESTS = 429
+    INTERNAL_SERVER_ERROR = 500
+    SERVICE_UNAVAILABLE = 503
+
+
+class AuthTokenBase(object):
```

Comment:
I got you point. 

`APNSAuthTokenFile` is kind of/has a `APNSAuthToken` rather than being is a `APNSAuthToken`. I therefore prefer each of them to acquire the behaviour through the inheritance( from the base class). Having said that, I didn't mean your approach is less correct. I am more than happy to take your approach.

Reply:
I understand about the base class thing but I'd prefer to have a smaller class hierarchy for now and not use abstract base classes.

Reply:
OK. Since you strongly prefer this way, I will address it shortly.